v2_test.go

```diff
@@ -102,6 +105,17 @@ func TestREADME(t *testing.T) {
 	require.Equal(t, m["alice"][0].Email, "alice@example.com")
 	require.Equal(t, m["alice"][0].Age, 29)
 
+	// JSON Response, check that we can execute the JSON received from query response
+	// DQL with JSON Data format should be valid as per https://docs.hypermode.com/dgraph/dql/json
+	jsonBytes, marshallErr := json.Marshal(m["alice"])
+	require.NoError(t, marshallErr)
+	mutationDQL = fmt.Sprintf(`{
+		"set": %s
+	  }`, jsonBytes)
+	resp, err = client.RunDQL(ctx, dgo.RootNamespace, mutationDQL, dgo.WithReadOnly())
```

Comment:
WithReadOnly?

Reply:
Ah thanks, copy and paste error. Fixed now but it gives the same error